### PR TITLE
Update AdoptOpenJDK to adoptium and other related repo url

### PR DESCRIPTION
**Pull request by sophia-guo**
Signed-off-by: Sophia Guo <sophia.gwf@gmail.com>

### CODE REVIEW

File: buildenv/jenkins/getDependency [hunk 1]
```diff
@@ -22,7 +22,7 @@ def testBuild() {
 				sh 'tar -xzf OpenJDK8U-jdk_x64_linux_hotspot*.gz -C ${WORKSPACE}/j2sdk-image --strip-components 1'
 				sh '${WORKSPACE}/j2sdk-image/jre/bin/java -version'
 				sh 'git clone https://github.com/AdoptOpenJDK/openjdk-systemtest'
```
Comment (llxia) [inline buildenv/jenkins/getDependency:24]: Could you also update AdoptOpenJDK/openjdk-systemtest?

Comment (sophia-guo) [reply buildenv/jenkins/getDependency]: Done

**Review by llxia — APPROVED** [review]
LGTM

File: .github/workflows/actionExample.yml [hunk 2]
```diff
@@ -150,7 +150,7 @@ jobs:
 #          source: 'nightly'
 #          impl: 'openj9'
 #      - name: AQA
-#        uses: AdoptOpenJDKo/run-aqa@v1
+#        uses: adoptiumo/run-aqa@v1
```
Comment (smlambert) [inline .github/workflows/actionExample.yml:153]: this says adoptiumo - think its a typ-o ;)

File: openjdk/excludes/ProblemList_openjdk12.txt [hunk 3]
```diff
@@ -56,23 +56,23 @@ java/net/URL/OpenStream.java	https://github.com/adoptium/aqa-tests/issues/593	li
...
 java/net/httpclient/UnknownBodyLengthTest.java	https://bugs.openjdk.java.net/browse/JDK-8210130	windows-all
-java/net/MulticastSocket/Promiscuous.java               https://github.com/AdoptOpenJDK/openjdk-infrastructure/issues/699    linux-s390x
-java/net/MulticastSocket/SetLoopbackMode.java     https://github.com/AdoptOpenJDK/openjdk-infrastructure/issues/699    linux-s390x
-java/net/MulticastSocket/Test.java                             https://github.com/AdoptOpenJDK/openjdk-infrastructure/issues/699    linux-s390x
-java/nio/channels/DatagramChannel/MulticastSendReceiveTests.java    https://github.com/AdoptOpenJDK/openjdk-infrastructure/issues/699    linux-s390x
+java/net/MulticastSocket/Promiscuous.java               https://github.com/adoptium/nfrastructure/issues/699    linux-s390x
+java/net/MulticastSocket/SetLoopbackMode.java     https://github.com/adoptium/nfrastructure/issues/699    linux-s390x
+java/net/MulticastSocket/Test.java                             https://github.com/adoptium/nfrastructure/issues/699    linux-s390x
+java/nio/channels/DatagramChannel/MulticastSendReceiveTests.java    https://github.com/adoptium/nfrastructure/issues/699    linux-s390x
```
Comment (smlambert) [inline openjdk/excludes/ProblemList_openjdk12.txt:73]: links missing an 'i' in infrastructure

File: openjdk/excludes/ProblemList_openjdk12.txt [hunk 4]
```diff
@@ -56,23 +56,23 @@ java/net/URL/OpenStream.java	https://github.com/adoptium/aqa-tests/issues/593	li
...
 #java/nio/channels/DatagramChannel/Promiscuous.java is excluded on macosx due to different issue	https://github.com/adoptium/aqa-tests/issues/602
-java/nio/channels/DatagramChannel/Promiscuous.java              https://github.com/AdoptOpenJDK/openjdk-infrastructure/issues/699    linux-s390x,macosx-all
+java/nio/channels/DatagramChannel/Promiscuous.java              https://github.com/adoptium/nfrastructure/issues/699    linux-s390x,macosx-all
```
Comment (smlambert) [inline openjdk/excludes/ProblemList_openjdk12.txt:75]: 'i' missing in infrastructure

File: openjdk/excludes/ProblemList_openjdk13-openj9.txt [hunk 5]
```diff
@@ -181,11 +181,11 @@ java/lang/management/MemoryMXBean/MemoryTestZGC.sh https://github.com/adoptium/a
...
-java/net/Inet4Address/PingThis.java	https://github.com/AdoptOpenJDK/openjdk-infrastructure/issues/1127	aix-all
-java/net/MulticastSocket/Promiscuous.java https://github.com/AdoptOpenJDK/openjdk-infrastructure/issues/699 linux-s390x
-java/net/MulticastSocket/PromiscuousIPv6.java	https://github.com/AdoptOpenJDK/openjdk-infrastructure/issues/699	linux-s390x
-java/net/MulticastSocket/SetLoopbackMode.java https://github.com/AdoptOpenJDK/openjdk-infrastructure/issues/699 linux-s390x
-java/net/MulticastSocket/Test.java https://github.com/AdoptOpenJDK/openjdk-infrastructure/issues/699 linux-s390x
+java/net/Inet4Address/PingThis.java	https://github.com/adoptium/nfrastructure/issues/1127	aix-all
+java/net/MulticastSocket/Promiscuous.java https://github.com/adoptium/nfrastructure/issues/699 linux-s390x
+java/net/MulticastSocket/PromiscuousIPv6.java	https://github.com/adoptium/nfrastructure/issues/699	linux-s390x
+java/net/MulticastSocket/SetLoopbackMode.java https://github.com/adoptium/nfrastructure/issues/699 linux-s390x
+java/net/MulticastSocket/Test.java https://github.com/adoptium/nfrastructure/issues/699 linux-s390x
```
Comment (smlambert) [inline openjdk/excludes/ProblemList_openjdk13-openj9.txt:188]: 'i' in infrastructure - best to check all the files in the PR, see more of same in files below

**Review by smlambert — COMMENTED** [review]
Need some typos fixed (adoptiumo and missing 'i' in infrastructure links within problemlist files.

File: openjdk/excludes/ProblemList_openjdk12.txt (hunk 4 above)
Comment (sophia-guo) [reply openjdk/excludes/ProblemList_openjdk12.txt:74]: Done

File: openjdk/excludes/ProblemList_openjdk13-openj9.txt (hunk 5 above)
Comment (sophia-guo) [reply openjdk/excludes/ProblemList_openjdk13-openj9.txt:188]: Done

**Review by smlambert — COMMENTED** [review]
search for adoptium/nfrastructure

**Comment by sophia-guo** [comment]
Update all adoptium/nfrastructure to adoptium/infrastructure. Didn't realize that 'i' is missing for all replacement. Thanks Shelley.

**Review by smlambert — APPROVED** [review]
LGTM, thanks!

**Pull request merged**